eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Rules/Utxo.hs

```diff
@@ -100,7 +100,7 @@ import Control.State.Transition.Extended
 import qualified Data.ByteString.Lazy as BSL (length)
 import Data.Coerce (coerce)
 import Data.Either (isRight)
-import Data.Foldable (foldl', sequenceA_, toList)
+import Data.Foldable as Foldable (foldl', sequenceA_, toList)
```

Comment:
Can you please rename all the qualifications in this PR from `Foldable` to `F`. This is needed for consistency, since that is what we use in other places where `Foldable` in imported qualified
```suggestion
import Data.Foldable as F (foldl', sequenceA_, toList)
```

Reply:
Done